paasta_tools/kubernetes_scripts/paasta_secrets_sync.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python
```

Comment:
paasta_tools/kubernetes/bin ?

Reply:
but why @robj?

Reply:
yeah, I got myself in a mess here. initially I went with `kubernetes/` and was hoping I could move some other stuff in a follow-up PR but it conflicts with the imported `kubernetes` package. I guess just put this in the top-level dir with everything else?

Reply:
just because that top level dir has *everything* in it - it'd be nice if we had a paasta_tools.kubernetes module in paasta-tools, with it's own bin dir